baremetal/metal3machine_manager.go

```diff
@@ -958,6 +1090,27 @@ func (m *MachineManager) setHostConsumerRef(ctx context.Context, host *bmh.BareM
 	}
 	host.OwnerReferences = hostOwnerReferences
 
+	m.Log.Info("Deleting nodeReuse label from BMH", host.Labels[NodeReuseLabelName])
```

Comment:
```suggestion
	m.Log.Info("Deleting NodeReuseLabelName from BMH", host.Labels[NodeReuseLabelName])
```

Reply:
It would be nice to be consistent with how we name hosts (ie BMH?BaremetalHost)

Reply:
Yes, went through the code and 'host' seems to be the most consistent in the context.